documentation/modules/operators/con-deleting-managed-topics.adoc

```diff
@@ -56,4 +56,31 @@ When the finalization tasks are successfully executed, the finalizer is removed
 
 Finalizers also prevent related resources from being deleted. 
 If the unidirectional Topic Operator is not running, it won't be able to remove the `metadata.finalizer`. 
-Consequently, an attempt to delete the namespace that contains the `KafkaTopic` resource won't complete until either the operator is restarted, or the finalizer is otherwise removed (for example using `kubectl edit`). 
\ No newline at end of file
+Consequently, an attempt to delete the namespace that contains the `KafkaTopic` resource won't complete until either the operator is restarted, or the finalizer is otherwise removed (for example using `kubectl edit`). 
+
+== Removing finalizers
+
+If you want to bypass the finalization process when deleting topics, you can remove the finalizers.   
+You might want to do this to delete topics more quickly; or to handle certain topics without invoking cleanup operations.
+Perhaps the cleanup operation associated with the finalization process is not completing as expected.
+
+To remove finalizers on all topics, use the following command:
+
+.Removing finalizers on topics
+[source,shell]
+----
+kubectl get kt -o=json | jq '.items[].metadata.finalizers = null' | kubectl apply -f -
+----
+
+The command uses the `jq` tool to modify the `KafkaTopic` (`kt`) resources by setting the finalizers to `null`.
+You can also use the command for a specific topic:
+
+.Removing a finalizer on a specific topic
+[source,shell]
+----
+kubectl get kt <topic_name> -o=json | jq '.metadata.finalizers = null' | kubectl apply -f -
+----
+
+After running the command, you can go ahead and delete the topics.
+
+WARNING: Be careful when removing finalizers, as any cleanup operations associated with the finalization process are not performed.
```

Comment:
Can we make this more specific?

Reply:
Thanks @tombentley -- is this the kind of thing:

WARNING: Be careful when removing finalizers, as any cleanup operations associated with the finalization process are not performed. 
For example, if you remove the finalizer from a `KafkaTopic` resource and subsequently delete the resource, the related topic in Kafka won't be deleted.